### ISSUE TITLE
Cannot read property 'length' of undefined (this.search)

### ISSUE THREAD

**Issue by antongorodezkiy** [monterail/vue-multiselect#58]
I've faced the following error:

```
vue.common.js:1019 [Vue warn]: Error when evaluating expression "filteredOptions.length === 0 && search.length": TypeError: Cannot read property 'length' of undefined (found in component: <multiselect>)
```

Changing `v-show="filteredOptions.length === 0 && search && search.length"` to `v-show="filteredOptions.length === 0 && search && search.length"` seems could fix it.

For now I just override the template.

**Comment by shentao** [comment]
Will look into it. Thanks! :)

**Comment by shentao** [comment]
Working on a new release right now. I will implement this fix there.
You can actually just write `v-show="filteredOptions.length === 0 && search"` and it works this way.

**Comment by antongorodezkiy** [comment]
Thank you!

**Comment by shentao** [comment]
@antongorodezkiy Fixed in 0.3.1
